catkit2/services/accufiz_interferometer/accufiz_interferometer.py

```diff
@@ -0,0 +1,270 @@
+import os
+import h5py
+import time
+import requests
+import uuid
+import numpy as np
+from scipy import ndimage
+import math
+from astropy.io import fits
+from glob import glob
+from catkit2.testbed.service import Service
+import os
+import threading
+
+
+
+def rotate_and_flip_image(data, theta, flip):
+    """
+    Rotate and/or flip the image data.
+
+    Args:
+        data (numpy.ndarray): Numpy array of image data.
+        theta (int): Rotation in degrees.
+        flip (bool): If True, flip the image horizontally.
+
+    Returns:
+        numpy.ndarray: Modified image after rotation and/or flip.
+    """
```

Comment:
This not exactly numpy docsrings. This would have to be stg like:

```
Parameters
----------
data : numpy.ndarray
    Numpy array of image data
theta : int
    Rotation in degrees.
flip : bool
    If True, flip the image horizontally

Returns
-------
numpy.ndarray
    Modified image after rotation and/or flip.
```

Reply:
Doh! Between you and me I have an AI copilot do doc string documentation for me so no worries.